snippets/javascript/class-object.md

```diff
@@ -0,0 +1,39 @@
+# Text encryptor
+
+You are making a text encryptor. It should take multiple words and output a
+combined version, where each word is separated by a dollar sign `$`.
+
+For example, for the words "hello", "how", "are", "you", the output should be
+`"$hello$how$are$you$"`.
+
```

Comment:
hey @AlinaTaoRao  can you give us some explanation about your code. Tell us about classes in js and how they work.

Reply:
`Classes` are a template for creating `objects`. They encapsulate data with code to work on that data.

This is an example of using a `class` to create `objects`. Make a text encryptor. It should take multiple words and output a combined version, where each word is separated by a dollar sign $. For example, for the words "hello", "how", "are", "you", the output should be "$hello$how$are$you$".